lib/osf-components/addon/components/osf-layout/left-nav-old/template.hbs

```diff
@@ -0,0 +1,9 @@
+<@gutters.left-gutter as |leftGutter|>
+    <SideNav @onLinkClicked={{@toggleSidenav}} as |sideNav|>
+        {{yield (hash
+            link=sideNav.link
+            leftGutterMode=leftGutter.mode
+            toggleSidenav=@toggleSidenav
+        )}}
+    </SideNav>
+</@gutters.left-gutter>
```

Comment:
What's `left-nav-old`?

Reply:
`left-nav-old` is a version of the left-nav that uses the old `side-nav` component. This will be phased out and replaced with just `left-nav`

Reply:
Does this PR need to update registries overview to use `leftNavOld`?